config/settings.yml

```diff
@@ -281,40 +281,7 @@ modules_appeals_api:
       # List of flags to indicate parts of documentation that are still WIP and not ready for production.
       # See the rswag:appeals_api:dev rake task, spec_helper, and DocHelpers#wip_doc_enabled? for usage info to opt-in.
       # NOTE: Do not remove this key even if it's empty.
-  higher_level_review_updater_enabled: true
   legacy_appeals_enabled: true
-  supplemental_claims_enabled: true
-  notice_of_disagreement_updater_enabled: true
-  supplemental_claim_updater_enabled: true
-  job_schedules:
```

Comment:
This should have been removed ages ago when this whole experiment to move our job schedules to Settings didn't work out. It's already been removed from the devops settings files.